app/controllers/projects_controller.rb

```diff
@@ -1,41 +1,47 @@
 class ProjectsController < ApplicationController
-  before_action :set_department, only: [:edit, :index, :new, :create]
-  load_and_authorize_resource through_association: :company
- 
+  load_and_authorize_resource :department
+  load_and_authorize_resource through: :department
+
+  def index
+    if !params[:checked].nil? && current_employee.role != 'Admin'
```

Comment:
same admin constant and  param name change 
and ! nil? and be replace with present